<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TauxChange</class>
 <widget class="QDialog" name="TauxChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(229, 188, 49);
font: italic 12pt &quot;Cambria&quot;;
background-color: rgb(33, 37, 41);</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: italic 25pt &quot;Cambria&quot;;
</string>
   </property>
   <property name="text">
    <string>Changement de taux</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: italic 15pt &quot;Cambria&quot;;
</string>
   </property>
   <property name="text">
    <string>A combien voulez vous changer le taux : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="NewTaux">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(229, 188, 49)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>New Val = </string>
   </property>
  </widget>
  <widget class="QLabel" name="NewVal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
